vtm/src/org/oscim/tiling/source/mapfile/MapDatabase.java

```diff
@@ -407,14 +407,10 @@ private void processBlock(QueryParameters queryParameters,
         }
     }
 
-    //    private long mCurrentRow;
-    //    private long mCurrentCol;
-
-    private void setTileClipping(QueryParameters queryParameters, long mCurrentRow, long mCurrentCol) {
-        long numRows = queryParameters.toBlockY - queryParameters.fromBlockY;
-        long numCols = queryParameters.toBlockX - queryParameters.fromBlockX;
```

Comment:
Can keep the calculations here since have already the `QueryParameters` as parameter.

Reply:
Thought it might be more efficient. But you can reverse it while merging, of course, if you want :)